scripts/create_cita_config.py

```diff
@@ -191,13 +191,24 @@ def template_create_from_arguments(self, args, contracts_dir_src,
         with open(executor_config, 'wt') as stream:
             toml.dump(executor_data, stream)
 
+        if args.stdout:
+            forever_config = os.path.join(self.configs_dir, 'forever.toml')
+            with open(forever_config, 'rt') as stream:
+                forever_data = toml.load(stream)
+                for process_data in forever_data['process']:
+                    process_data['args'].append('-s')
+            with open(forever_config, 'wt') as stream:
+                toml.dump(forever_data, stream)
+
         jsonrpc_config = os.path.join(self.configs_dir, 'jsonrpc.toml')
         with open(jsonrpc_config, 'rt') as stream:
             jsonrpc_data = toml.load(stream)
             jsonrpc_data['http_config']['listen_port'] \
                 = str(args.jsonrpc_port)
             jsonrpc_data['ws_config']['listen_port'] \
                 = str(args.ws_port)
+            if args.enable_version:
+                jsonrpc_data['enable_version'] = True
```

Comment:
I am confused the default `enable_version` is false or true?

Several steps this config works:

1. When cita compiled, this config located in  `target/install/scripts/config_tool/config_example`

2. When nodes created, the first step is copying config in scripts dir to template dir, 

3. When nodes created, the second step is rewriting config in template dir to node dir.

In your implecation, you set `enable_version` false ` in step 1, then set it true in step 2.

Reply:
The default enable_version is false.
In step 2, only if --enable_version is present when we run create_cita_config.py, enable_version in jsonrpc.toml would be set to true.

Reply:
ok